real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/AngleGraph.java

```diff
@@ -51,40 +27,50 @@ public void updateGraph(int angle) {
 	private class Graph extends JPanel {
 		private static final long serialVersionUID = 6015150544448011207L;
 
-		private static final int WIDTH = 200;
-		private static final int HEIGHT = 200;
+		private int graphWidth = 200;  //200 is the default  value for the width
+		private int graphHeight = 200; //200 is the default value for the height
 		private int endx;
 		private int endy;
-		private static final int OFFSET = 10;
-		private static final int startx = WIDTH/2 + OFFSET;
-		private static final int starty = HEIGHT/2 + OFFSET;
-		private static final double RADIUS = (double)WIDTH/2;
+		private int offset;
+		private int start_x;
+		private int start_y;
+		private double RADIUS;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [The field name indicates a constant but its modifiers do not](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1867203118/issues/source?bid=2938665&fileBranchId=3059924#l37)